README.md

```diff
@@ -1,4 +1,255 @@
+    cpu: "16"
+    memory: "8Gi"
+  limits:
+    memory: "8Gi"
+```
+
+### Create the StatefulSet
+
+Apply `example.yaml`:
+
+```
+kubectl create -f example.yaml
+```
+
+Check that the pods were created:
+
+```
+kubectl get pods
+```
+
+```
+NAME                                  READY   STATUS    RESTARTS   AGE
+cockroach-operator-6f7b86ffc4-9t9zb   1/1     Running   0          3m22s
+cockroachdb-0                         1/1     Running   0          2m31s
+cockroachdb-1                         1/1     Running   1          102s
+cockroachdb-2                         1/1     Running   0          46s
+```
+
+Each pod should have `READY` status soon after being created.
+
+## Access the SQL shell
+
+Launch a pod that runs indefinitely with the `cockroach` binary. <!-- Depending on how you authenticated the cluster: -->
+
+- Kubernetes CA:
+
+	```
+	kubectl create -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/examples/client-secure.yaml
+	```
+
+	Approve the certificate signing request generated by Kubernetes:
+
+	```
+	kubectl certificate approve default.client.root
+	```
+	
+<!-- - Non-Kubernetes CA:
+
+	If you didn't use `cockroach cert` to generate the client certificate, substitute an appropriate `secretName` before applying this file:
+
+	```
+	kubectl create -f https://raw.githubusercontent.com/cockroachdb/cockroach-operator/master/examples/client-secure-byoc.yaml
+	``` -->
+
+You can then access the SQL shell and run SQL statements from this pod:
+
+```
+kubectl exec -it cockroachdb-client-secure -- ./cockroach sql --certs-dir=/cockroach-certs --host=cockroachdb-public
+```
+
+If you deployed an insecure cluster, you can access the SQL shell directly:
+
+```
+kubectl run cockroachdb -it --image=cockroachdb/cockroach:v20.1.4 --rm --restart=Never -- sql --insecure --host=cockroachdb
+```
+
+If you want to [access the Admin UI](#access-the-admin-ui), create a SQL user with a password while you're here:
+
+```
+CREATE USER roach WITH PASSWORD 'Q7gc8rEdS';
+```
+
+```
+\q
+```
+
+## Access the Admin UI
+
+To access the cluster's [Admin UI](https://www.cockroachlabs.com/docs/v20.1/admin-ui-overview), port-forward from your local machine to one of the pods:
+
+```
+kubectl port-forward cockroachdb-0 8080
+```
+
+Access the Admin UI at `https://localhost:8080`. Note that [certain Admin UI pages](https://www.cockroachlabs.com/docs/v20.1/admin-ui-overview#admin-ui-access) require `admin` access.
+
+## Configure the cluster
+
+You can configure the cluster by modifying and reapplying `example.yaml`.
+
+<!-- ### Expand disk size
+
+To expand persistent volumes, edit `resources.requests.storage`:
+
+```
+resources:
+  requests:
+    storage: 100Gi
+```
+
+Apply the new configuration:
+
+```
+kubectl apply -f example.yaml
+``` -->
+
+### Scale the cluster
+
+To scale the cluster, edit `nodes`:
+
+```
+nodes: 4
+```
+
+Do **not** scale down to fewer than 3 nodes. This is considered an anti-pattern and will cause errors.
```

Comment:
Please explain a bit more that this is a restriction of cr, and not of the operator.